### PR TITLE
testing/traefik: new aport

### PR DESCRIPTION
**Pull request by joeholden**
As per the title, new port for Traefik - build and runtime tested.

Not sure what else you guys need other than a PR but reviews would be appreciated.

### CODE REVIEW

File: testing/traefik/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,60 @@
+pkgname=traefik
+pkgver=1.7.4
+pkgrel=7
```
Comment (andypost) [inline testing/traefik/APKBUILD]: why 7 instead of 0?

**Comment by joeholden** [comment]
Yup will do

File: testing/traefik/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+pkgdesc="The Cloud Native Edge Router"
+url="https://traefik.io"
+arch="all"
+license="MIT"
+makedepends="go go-bindata"
+install="$pkgname.pre-install"
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+subpackages="$pkgname-openrc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/containous/$pkgname/archive/v$pkgver.tar.gz
```
Comment (andypost) [inline testing/traefik/APKBUILD]: please change it to `source="$pkgname-$pkgver.tar.gz::https://github.com/containous/traefik/archive/v$pkgver.tar.gz` to make navigation easy

PS: https://github.com/containous/traefik/releases/tag/v1.7.5 is out

Comment (joeholden) [reply testing/traefik/APKBUILD]: Ok I'll update and fix that, thanks

Comment (joeholden) [reply testing/traefik/APKBUILD]: done, updated to 1.7.5 also

**Pull request closed without merging**